card/card.css

```diff
@@ -0,0 +1,256 @@
+}
+
+* {
+   margin: 0;
+   padding: 0;
+   box-sizing: border-box;
+}
+
+body {
+   /* display: flex; */
+}
+
+.nav-container {
+   display: flex;
+   width: 100%;
+   height: 10vh;
+   position: fixed;
+   top: 0;
+   background-color: var(--primary-color);
+   justify-content: space-around;
+   align-items: center;
+   z-index: 1;
+}
+
+.nav-container a:hover {
+   color: rgb(247, 75, 75);
+}
+
+main {
+   width: 100%;
+   display: flex;
+}
+
+aside {
+   width: 20%;
+   background-color: var(--footer-black);
+   height: 90vh;
+   position: sticky;
+   top: 10vh;
+   height: 90vh;
+   overflow-y: auto;
+}
+
+.div {
+   background-color: var(--secondary-color);
+   width: 80vw;
+   color: white;
+}
+
+.sidebar-list {
+   padding: 0.8rem;
+   list-style: none;
+   text-decoration: none;
+}
+
+.sidebar-list-header {
+   padding: 1rem 0.6rem;
+   font-weight: bold;
+   font-size: 2vh;
+   color: white;
+}
+
+.sidebar-list a {
+   text-decoration: none;
+   transition: all 0.3s cubic-bezier(0, 0.76, 1, 1);
+}
+
+.sidebar-list a:hover {
+   padding: 1rem;
+}
+
+.nav-container-link {
+   display: flex;
+   text-decoration: none;
+   color: var(--secondary-color);
+}
+
+.link {
+   color: white;
+}
+
+.sidebar-link {
+   font-family: "Source Sans Pro", sans-serif;
+   text-decoration: none;
+   color: white;
+   font-size: large;
+}
+
+.sidebar-link:hover {
+   color: rgb(153, 197, 255);
+}
+
+.aside li:hover {
+   background-color: #ececec;
+}
+
+.card {
+   display: flex;
+   border: 1px solid var(--primary-color);
+   padding: 3rem;
+   width: 400px;
+   height: 30vh;
+
+   border-color: var(--primary-color);
+}
+
+.btn-primary {
+   padding: 0.7rem;
+
+   /* background-color: var(--primary-color); */
+   margin-top: 2rem;
+   background-color: var(--primary-color);
+   color: white;
+   outline: none;
+   border: none;
+   border-radius: 4px;
+   cursor: pointer;
+}
+
+.btn-primary:hover {
+   background-color: #001224;
+}
+
+.card-title {
+   padding-bottom: 1rem;
+   font-size: 1rem;
+}
+
+.text-center {
+   text-align: center;
+}
+
+.text-end {
+   display: flex;
+   text-align: right;
+   justify-content: flex-end;
+}
+
+.parent-container {
+   width: 80%;
+   display: flex;
+   flex-direction: column;
+   /* background-color: var(--footer-black); */
+}
+
+iframe {
+   padding: 3rem;
+}
+
+.card-section {
+   padding: 3rem;
+   display: flex;
+   margin-top: 5rem;
+}
+
+.product-card-section {
+   padding: 3rem;
+}
+
+img {
+   height: 100%;
+   width: 100%;
+   object-fit: contain;
+   object-position: center;
+}
+
+.cardSecondary {
+   display: flex;
+   justify-content: center;
+   flex-direction: column;
+   border: 2px solid var(--primary-color);
+   min-height: 30vh;
+   width: 20vw;
+
+   cursor: pointer;
+}
+
+.cardSecondary:hover {
+   box-shadow: 11px 15px 20px -3px rgba(0, 0, 0, 0.74);
+   -webkit-box-shadow: 11px 15px 20px -3px rgba(0, 0, 0, 0.74);
+   -moz-box-shadow: 11px 15px 20px -3px rgba(0, 0, 0, 0.74);
+}
```

Comment:
The hover effect is too sharp.
Maybe add the transition property for a smooth hover effect.